src/d3d/nvapi_d3d_low_latency_device.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "../nvapi_private.h"
+#include "../shared/shared_interfaces.h"
+#include "../util/com_pointer.h"
+
+namespace dxvk {
+    class NvapiD3dLowLatencyDevice {
+      public:
+        static bool SupportsLowLatency(IUnknown* device);
+        static bool LatencySleep(IUnknown* device);
+        static bool SetLatencySleepMode(IUnknown* device, bool lowLatencyMode, bool lowLatencyBoost, uint32_t minimumIntervalUs);
+        static bool GetLatencyInfo(IUnknown* device, D3D_LATENCY_RESULTS* latencyResults);
+        static bool SetLatencyMarker(IUnknown* device, uint64_t frameID, uint32_t markerType);
+
+      private:
+        inline static std::unordered_map<IUnknown*, ID3DLowLatencyDevice*> m_lowLatencyDeviceMap;
```

Comment:
Sorry for seeing this that late, could you please also add a `ClearCacheMaps` function like https://github.com/jp7677/dxvk-nvapi/blob/master/src/d3d12/nvapi_d3d12_device.h#L31 and call it in the same places. This mostly prevents caching issues for the tests by always starting with a clean environment.

Reply:
I added the ClearCacheMaps function for the D3dLowLatencyDevice interface.